lib/src/main/java/graphql/nadel/validation/NadelTypeValidation.kt

```diff
@@ -270,19 +255,17 @@ internal class NadelTypeValidation {
             }
 
         // If it can be reached by using your service, you must own it to return it!
-        val referencedTypes = getReachableTypeNames(engineSchema, service, definitionNames + matchingImplementsNames)
-
-        return (definitionNames + referencedTypes).toSet()
+        return getReachableTypeNames(service, definitionNames + matchingImplementsNames)
```

Comment:
Turns out `getReachableTypeNames` is a superset of,

```
return (definitionNames + referencedTypes).toSet()
```

So it was useless to add them.